src/client/app/App.jsx

```diff
@@ -1,26 +1,62 @@
 'use strict';
 
 import React, { Component } from 'react';
-import { Route, Switch } from 'react-router-dom';
+import { Route, Switch, Redirect } from 'react-router-dom';
 
 import Home from './components/landing/Home';
 import NotFound from './components/NotFound';
-import Profile from './components/dashboard/Profile';
-import Add from './components/add-child/Add';
+import Dashboard from './components/dashboard/Dashboard';
 import Calendar from './components/calendar/Calendar';
+import AddKiddo from './components/add-child/AddKiddo';
+import ApiHelper from './utils/apiHelper';
 
 import '../index.css';
 
 class App extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      user: null
+    };
+    this.saveNewKiddo = this.saveNewKiddo.bind(this);
+    this.getUser = this.getUser.bind(this);
+  }
+
+  componentDidMount () {
+    this.getUser();
+  }
+
+  getUser () {
+    let user = ApiHelper().getCurrentUser();
```

Comment:
ApiHelper should be called and loaded into a var before using instead of 
`ApiHelper().get()` have `Db.get()`